src/axom/mint/docs/sphinx/sections/getting_started.rst

```diff
@@ -0,0 +1,275 @@
+
+  #. ``xargs::coords``:
+     Indicates that in addition to the index of the cell, ``cellIdx``, the
+     supplied kernel takes two additional arguments:
+
+      a. ``coords``, a matrix that stores the node coordinates of the cell, and
+      b. ``nodeIds``, an array that stores the IDs of the constituent cell nodes.
+
+.. note::
+
+   Since in this kernel the ``nodeIds`` are not used, the third kernel argument
+   is annotated with the ``AXOM_NOT_USED`` macro to silence compiler warnings.
+
+The cell node coordinates matrix, defined by ``axom::numerics::Matrix`` is
+organized such that:
+
+  * The number of rows of the matrix corresponds to the cell dimension, and,
+  * The number of columns of the matrix corresponds to the number of cell nodes.
+  * The :math:`ith` column vector of the matrix stores the coordinates of the
+    :math:`ith` cell node.
+
+In this example, the 2D :ref:`UniformMesh`, consists of 2D rectangular cells,
+which are defined by :math:`4` nodes. Consequently, the supplied node coordinates
+matrix to the kernel, ``coords``, will be a :math:`2 \times 4` matrix of the
+following form:
+
+.. math::
+   :nowrap:
+
+   \[ \left(
+   \begin{array}{ c c c c }
+      x_0 & x_1 & x_2 & x_3 \\
+      y_0 & y_1 & y_2 & y_3
+   \end{array}
+   \right) \]
+
+Within the body of the kernel, the centroid of the cell is calculated by
+averaging the node coordinates. The code loops over the columns of the
+``coords`` matrix (i.e., the cell nodes) and computes the sum of each
+node coordinate in ``xsum`` and ``ysum`` respectively. Then, the average is
+evaluated by multiplying each coordinate sum with :math:`1/4`. The result is
+stored in the corresponding field array, ``xc``, which, is captured by the
+`Lambda Expression`_.
+
+Since, the cell centroid is a 2D vector quantity, each cell entry has an
+x-component and y-component. Multi-component fields in Mint are stored using a
+2D row-major contiguous memory layout, where, the number of rows corresponds to
+the number of cells, and the number of columns correspond to the number of
+components of the field. Consequently, the x-component of the centroid of a cell
+with ID, ``cellIdx`` is stored at ``xc[ cellIIdx * NUM_COMPONENTS  ]`` and the
+y-component is stored at ``xc[ cellIdx * NUM_COMPONENTS + 1 ]``, where
+``NUM_COMPONENTS=2``.
+
+.. _step6:
+
+Step 6: Output the Mesh in VTK
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Last, the resulting mesh and data can be output in the Legacy `VTK File Format`_,
+which can be visualized by a variety of visualization tools, such as, `VisIt`_
+and `ParaView`_ as follows:
+
+.. literalinclude:: ../../../examples/mint_uniform_mesh.cpp
+   :start-after: sphinx_tutorial_walkthrough_vtk_start
+   :end-before: sphinx_tutorial_walkthrough_vtk_end
+   :language: C++
+   :linenos:
+
+The resulting VTK file is stored in the specified file, ``uniform_mesh.vtk``.
+A depiction of the mesh showing a plot of `Himmelblau's Function`_ computed
```

Comment:
Please move this sentence and the figure to Step 7, after the Note.  Running the example produces the `.vtk` file, which you then show off in the figure.